tests/opentelemetry-test-utils/src/opentelemetry/test/test_base.py

```diff
@@ -85,6 +90,61 @@ def sorted_spans(self, spans):  # pylint: disable=R0201
             reverse=True,
         )
 
+    def sorted_metrics(self, metrics):  # pylint: disable=R0201
```

Comment:
maybe better name would be `sort_metrics` or `get_sorted_metrics` as according to the conventions is has to start with verb :)

Reply:
You're right, but I want to use the same convention in this class
for metrics like spans
and the function to sort spans is sorted_spans

Reply:
ok LGTM :)